managing-cf/audit-events.html.md.erb

```diff
@@ -0,0 +1,214 @@
+---
+title: Audit Events
+owner: CAPI
+---
+
+<strong><%= modified_date %></strong>
+
+## Overview
+Audit events help Cloud Foundry operators monitor actions taken against resources (such as applications) via
+user or system actions.
+
+### Differences from other types of events
+Despite being similar in name, audit events are different from [Cloud Foundry Usage Events](./usage-events.html).
+Unlike usage events, audit events are **not** suitable for application and service billing.
+
+Another tool to help administrators monitor user actions on Cloud Foundry is [Security Event Logging](../../loggregator/cc-uaa-logging.html).
+Unlike Security Event Logs, audit events do **not** cover all API endpoints, but provide more details about a user's actions.
+
+
+### When are events created?
```

Comment:
Let's re-order some of this to try and get the meat near the top

Overview
- differences
Using
- all the stuff in it
Event creation
Considerations

Reply:
I left the "Viewing Audit Events" part in the Overview section and moved the rest as suggested.
https://github.com/cloudfoundry/docs-running-cf/pull/60/commits/04694449662d6e0cbf66445e09770204cb416424#diff-ba05cd371854e68a073991abb3da83e4R190